Remotion/Core/Reflection.CodeGeneration/CustomPropertyEmitter.cs

```diff
@@ -70,7 +70,7 @@ public Type[] IndexParameters
       get { return _indexParameters; }
     }
 
-    public IMethodEmitter GetMethod
+    public IMethodEmitter? GetMethod

```

Comment:
interesting. that's just...bizarre. not sure though, if it's also a null-bug in the design.

Reply:
not with the current implementation differences, we can't support a symmetrical API that's also safe. this difference might well be a bug and the get and set method be not-null. it's also possible that there's a subtle difference between event accessors and property accessors. either way, we can't repair this here and we can't fudge on the api without making it buggier. so, yes, prperty and event will be different until someone figures out if this is correct and fixes it in the correct order. given that Reflection.CodeGen is a legacy assembly, that won't happen anytime soon :D